schemas/json/layout/layout.schema.v1.json

```diff
@@ -427,6 +427,40 @@
         }
       }
     },
+    "radioAndCheckboxComponents": {
```

Comment:
Intead of a `radioAndCheckboxComponents` where we duplicate all properties in `selectionComponents` I think we should "extend" the selectionComponents using `allOf`. Something like this:
```
"radioAndCheckboxComponents": {
      "allOf": [{ "$ref": "#/definitions/selectionComponents" }],
      "properties": {
        "layout": {
          "type": "string",
          "enum": ["column", "row"],
          "title": "Layout",
          "description": "Define the layout style for the options"
        }
      }

Reply:
Nice, that is a lot better!